.github/workflows/api-v1-production-build.yml

```diff
@@ -1,4 +1,4 @@
-name: Production Build
+name: Production Builds
```

Comment:
Changed this name so all of the production builds we run get lumped into the same group visually.